initramfs/init

```diff
@@ -58,6 +58,12 @@ gnubee_boot(){
 	mdadm --assemble --scan --auto=md --run --freeze-reshape
    fi
 
+   if test -x /bin/lvm; then
```

Comment:
wouldn't it be possible to check for udev here too ? to avoid the issues referenced in the conversation ?

Reply:
It wouldn't help.  We know that udev won't be installed on the initramfs, so there is nothing to check for.
We need to either:
1/ install udev and make sure it works and doesn't break anything
2/ fix lvm so that is doesn't wait so much.

Reply:
Would this work?
https://wiki.gentoo.org/wiki/Custom_Initramfs#LVM
I don't have much time now to fiddle with my gnubee (some work projects demand much of my attention span these days)

Reply:
Yes, that seems to work.  Thanks.
I added the suggested lvm.conf file and now I don't get the long delay.
I've build firmware based on Linux-5.7.2 and uploaded it.
Please test and confirm that it assembled lvm volumes properly.

Reply:
I'm trying to compile the new firmware, but I'm hitting something stupid that is puzzling me... When I do `../gnubee-tools/scripts/gbmake firmware gbpc1-5.7` (even as root), all I get is "permission denied". But, the file has all permissions ...
`-rwxrwxr-x 1 vic vic 5486 Jun 15 18:01 ../gnubee-tools/scripts/gbmake`

Any ideas accepted :-) I'm eager to test the new version.

Reply:
Try running it as
`  bash -x ../gnubee-tools/scripts/gbmake firmware gbpc1-5.7`

Reply:
It was something really stupid: the filesystem was mounted by my user via /etc/fstab, so it was mounted _noexec_ by default. As soon as I added _exec_ to the options for the filesystem mount, all started working.
Now I have to find why it insists in compiling 4.4.14 ... but that is refreshing something I did too long ago :-)

Reply:
Some cleanup (actually a full "git clone" anew :-)) fixed the problems, and the Gnubee is happily humming compiling its firmware.

Reply:
Tested! Works like a charm! LVM initialization from the initrd is almost instantaneous. Now, my GnuBee1 runs kernel 5.7.2+. Now I have to get acquainted with the changes in the new kernel, specially the network interfaces.

Thanks @neilbrown :-)